ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationSidebar.tsx

```diff
@@ -80,7 +88,20 @@ function NavigationSidebar({
                     <LeftNavItem
                         isActive={location.pathname.includes(networkBasePathPF)}
                         path={networkBasePathPF}
-                        title={basePathToLabelMap[networkBasePathPF]}
+                        title={
+                            <Flex>
+                                <FlexItem>Network Graph</FlexItem>
+                                <FlexItem>
+                                    <Badge
+                                        style={{
+                                            backgroundColor: 'var(--pf-global--palette--cyan-400)',
+                                        }}
+                                    >
+                                        2.0 preview
+                                    </Badge>
+                                </FlexItem>
+                            </Flex>
+                        }
```

Comment:
i moved this directly here because the `basePathToLabelMap` was meant to just be a string path to a string label conversion 😅